SqlServerLib/SQLServer.cpp

```diff
@@ -0,0 +1,139 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check it.
+
+// PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+
+
+#include "pch.h"
+#include "framework.h"
+#include "SQLServer.h"
+
+
+SQLServer::SQLServer(const ConnectParams& connect_string)
+{
+	LOG = LOG->GetInstance();
+	Connect(connect_string);
+}
+
+SQLServer::~SQLServer()
+{
+	Disconnect();
+}
+
+
+bool SQLServer::Connect(const ConnectParams& connect_string)
+{
+	bool check_connection = m_connection.isConnected();
+
+	if (!check_connection)
+	{
+		m_params = connect_string;
+		std::string con = connect_string.server + "@" + connect_string.database;
+		m_connection.Connect(_TSA(con.c_str()), _TSA(connect_string.username.c_str()), _TSA(connect_string.password.c_str()), SA_SQLServer_Client);
+		LOG_INFO << "Connected to SQL Server...\n";
+	}
+
+	return m_connection.isConnected();
+}
+
+void SQLServer::InsertEmail(const Email& email)
+{
+	const std::string TABLE_NAME{ "Users" };
+	const std::string COLUMN_NAME_ID{ "Email_address" };
+	const std::string COLUMN_NAME_ADDRESS{ "Password" };
+	
+
+	const std::string INSERT_COMMAND{ "INSERT INTO " };
```

Comment:
maybe make all commands as const global vars instead of creating those strings in each func